### DISCUSSION TITLE
email defaults. Edit! Solved, in config file

### DISCUSSION THREAD

**Discussion by Evehic** [mastodon/mastodon#21260] [General]
When setting up a server the global default email settings for all users signing up are:
 - email if mentioned
 - email if followed

This causes a lot of emails to be sent from the client. An issue if there is a quota.

Edit: Fixed by editing fields in config file and restartung

**Comment by abdessalaam** [comment]
Could you please share where is the file, and what edits you made? This would be useful. Thanks!
